.github/workflows/production-heartbeat.yml

```diff
@@ -0,0 +1,73 @@
+name: production-heartbeat
+on:
+  schedule:
+    # Times are UTC
+    - cron: '45 1,5,9,13,17,21 * * *'
+jobs:
+  production-heartbeat:
+    strategy:
+      # By default, if any job in a matrix fails, all other jobs are immediately cancelled. This makes the jobs run to completion instead.
+      fail-fast: false
+      matrix:
+        os: [{vm: ubuntu-latest, exe: .sh}, {vm: windows-2019, exe: .cmd}]
+        node: ['lts/*']
+    runs-on: ${{ matrix.os.vm }}
+    steps:
+      # === Setup. We need to get the code, set up nodejs, and create the results directory. ===
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node }}
+      - run: mkdir smoke-test-results
+      # === Make three attempts to install sfdx through npm ===
+      - name: Install SFDX
+        id: sfdx_install
+        # Sleep for 5 seconds between attempts, since it's possible that problems are network-related.
+        run: npm install -g sfdx-cli || (sleep 5 && npm install -g sfdx-cli) || (sleep 5 && npm install -g sfdx-cli)
```

Comment:
I think it would be good to increase the timeout on the second sleep.

Reply:
Fair enough. What if we did an increasing backoff strategy, where the first timeout was 600 seconds (10 minutes), and then the second was 1800 seconds (30 minutes)? Or something similar but with different numbers?